assets/src/blocks/Cookies/CookiesFrontend.js

```diff
@@ -70,46 +70,46 @@ export const CookiesFrontend = props => {
 
   const updateConsent = (key, granted) => {
     dataLayer.push({
-      'event' : 'updateConsent'
+      event: 'updateConsent',
     });
 
     if (!ENABLE_GOOGLE_CONSENT_MODE) {
       return;
     }
 
     gtag('consent', 'update', {
-      [key]: granted ? 'granted' : 'denied',
+      [ key ]: granted ? 'granted' : 'denied',
     });
     dataLayer.push({
-      'event': 'updateConsent',
-      [key]: granted ? 'granted' : 'denied',
+      event: 'updateConsent',
+      [ key ]: granted ? 'granted' : 'denied',
```

Comment:
What was the main motivation to change `[key]` to `[ key ]` but `{ key }` to `{key}`?

Reply:
Thank you for pointing this out, I will take a look at it.